rascal2/ui/model.py

```diff
@@ -95,3 +99,22 @@ def load_r1_project(self, load_path: str):
         self.project = RAT.utils.convert.r1_to_project_class(load_path)
         self.controls = RAT.Controls()
         self.save_path = str(Path(load_path).parent)
+
+    def update_project_general_settings(
+        self, calculation: Calculations, model: LayerModels, geometry: Geometries
+    ) -> None:
+        """Updates the project general settings.
+
+        Parameters
+        ----------
+        calculation : Calculations
+            The updated calculation of the project.
+        model : LayerModels
+            The updated model of the project.
+        geometry : Geometries
+            The updated geometry type of the project.
+        """
+        self.project.calculation = calculation
+        self.project.model = model
+        self.project.geometry = geometry
+        self.update_project_view.emit()
```

Comment:
in fact we probably also want it to capture any validation errors and propagate them back to the widget so that they can show up to the user if they try to save an invalid project: see similar behaviour in the Controls widget: 

![Screenshot from 2024-08-19 15-19-12](https://github.com/user-attachments/assets/37575904-33e6-4580-bbee-8700c05df15a)

Reply:
About the validation, I think it should definitely be there but I don't think it is with in the scope of the issue as it is defined. So, I am not going to make any other changes here and prioritise merging this and then whoever makes the tabs can add the validation.